File: Day 25/day25.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int t;
    cin>>t;

    while(t--)
    {
        int n;
        cin>>n;
        int c = 0;
        for(int i=2; i<=sqrt(n); i++)
        {
            if(n%i==0)
                c++;
        }

        if(n==1)
        {
            cout<<"Not prime"<<endl;
            continue;
        }
        if(c==0)
            cout<<"Prime"<<endl;
        else
            cout<<"Not prime"<<endl;
    }
    return 0;
}

File: Day 25/day25.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import math
t = int(input())

while t>0:

    n = int(input())
    c = 0
    for i in range(2, math.floor(math.sqrt(n))+1):
        if n%i==0:
            c += 1
    if n==1:
        print("Not prime")
        t -= 1
        continue
    if c==0:
        print("Prime")
    else:
        print("Not prime")
    t -= 1


